### PR TITLE
Use the right package name for the install size badge

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,4 +134,4 @@ Base Log SVG by [Freepik](http://www.freepik.com/) from [www.flaticon.com](http:
 
 [CONTRIBUTING](./.github/CONTRIBUTING)
 
-[LICENSE (Mozilla Public License)](./LICENSE)
```
Comment (shellscape) [inline README.md]: removed line break?

**Comment by shellscape** [comment]
Thanks for the PR. Please check the line break at the end of the file, looks like your editor might have done something there.

**Comment by shellscape** [comment]
Thanks for brining the issue to my attention. I looked in on the repo and there were some other housekeeping tasks needed (and some more replacements needed) so I just did that all in one swoop. `master` should be correct now

**Comment by ForsakenHarmony** [comment]
just edited it on github, no idea what happened there 🙈

**Pull request closed without merging**